docs/UserGuide.md

```diff
@@ -63,189 +63,149 @@ navigation of modules.
 
- * User can add any module by specifying their desired Module Code (eg. CS1231),
- Modular Credits (eg. 4), and Tags (eg. Core) along with the `add` command, in their
- input.
- 
- * Module Code and Modular Credit fields are compulsory inputs but Tag fields are optional.
- Multiple Tags are allowed per module. Duplicate Modules (Modules with all the same fields)
- are not allowed.
- 
- * When you add a module, GradPad automatically gives it its correct module title based on the module code you've
- entered. For example, Gradpad automatically gives the CS2103T module its title, "Software Engineering".
- As such, there is no need to enter module titles yourself.
-
-Format: `add c/ {module code} cr/ {modular credits}`
+To add the module:
 
-Example of usage: `add c/CS2103T cr/4`
+1. Type `add c/cs2101 cr/4 t/superbmodule t/superfun` into the command box, and press **Enter** to execute it.<br>
+![Add1](images/Add1.png) 
 
-Expected outcome:
+2. The result display box will display the message "New module added", along with the details of the module. You will then be able to see the CS2101 module at the bottom of the Completed Modules list.<<br>
+![Add2](images/Add2.png)
 
-<img src="images/add.png" width="550px" height="350px">
+### Viewing all added modules: `list`
 
-### View Added Modules : `list`
+You can view all the modules you have added into the Completed Modules list in GradPad using this command. This is especially useful when you have previously used a command that filters the list (i.e. `find`) and you wish to restore the list to its original state.
 
-Description:
-
- * Displays the list of Current Modules in GradPad in the `Current Modules` section.
- 
- * Users can use this command to reset the view of the `Current Modules`
- section, as certain commands (Eg. find) might alter the 
- view of the `Current Modules` section.
+Example:<br>
+You have filtered the Completed Modules list to show modules whose module codes contain 'cs1', and you wish to restore the list to view all the modules you have added.
 
-Format: `list`
+To view all modules: 
 
-Example of usage: `list`
+1. Type `list` into the command box, and press **Enter** to execute it.
+![List1](images/List1.png)
 
-Expected outcome:
+2. The result display box will display the message "Listed all modules" and you will be able to view all your previously added modules in the Completed Modules list.
+![List2](images/List2.png)
 
-<img src="images/list.png" width="550px" height="350px">
+### Editing module details: `edit`
 
-### Edit a Module: `edit`
+This command allows you to edit the details of a module that you have added into GradPad.
 
-Description:
+You can edit a module by specifying the index of the module in the Completed Modules list, followed by the details of the modules you wish to edit with their respective prefixes, 'c/' for Module Code, 'cr/' for Modular Credits, etc.
```

Comment:
Suggestion: Should we explain that we allow users to edit multiple fields in the same command?

Reply:
ok will address this! thanks bro

Reply:
done!